core/core.go

```diff
@@ -205,10 +209,7 @@ func NewNode(config NodeConfig) (*TextileNode, error) {
 	// determine the best routing
 	var routingOption core.RoutingOption
 	if config.IsMobile {
-		// TODO: Determine best value for this setting on mobile
-		// cfg.Swarm.DisableNatPortMap = true
-		// NOTE: trying normal routing for a bit
-		routingOption = core.DHTOption
+		routingOption = core.DHTClientOption
```

Comment:
been testing with this now that I think I understand the difference... in this mode, the node doesn't respond to node search requests. on mobile, the node is hardly ever awake, so seems kinda pointless. we should get some battery savings

Reply:
nic